runtime/src/chpl-cache-support.c

```diff
@@ -56,33 +57,6 @@ static inline void saturating_add(saturating_count_t *x, unsigned int num) {
   *x = v;
 }
 */
-// If we have a mask representing 2^n - 1,
```

Comment:
Is this actually a related change (to the arg bundle stuff)?  Or is it an unrelated change that's gotten swept up in your branch?